tests/workflows/snvphylnfc.nf.test

```diff
@@ -70,7 +70,7 @@ nextflow_workflow {
 
         when {
             params {
-                input = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/409864a836dd4c2cd6191e7456881b973ca04748/assets/samplesheet.csv"
+                input = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv"
```

Comment:
I updated these as part of the PR, since the new samplesheet is now included in dev.